lib/backend/firestore/firestorebk.go

```diff
@@ -138,6 +141,8 @@ type record struct {
 	Value      []byte `firestore:"value,omitempty"`
 	RevisionV2 string `firestore:"revision,omitempty"`
 	RevisionV1 string `firestore:"-"`
+
+	snapShot *firestore.DocumentSnapshot `firestore:"-"`
```

Comment:
Is this addition just to include the objectively wrong behavior of `GetRange` in `Items`?